gcloud-java-storage/src/main/java/com/google/gcloud/storage/Bucket.java

```diff
@@ -179,8 +179,8 @@ public Blob get(String blob, BlobSourceOption... options) {
    * @throws StorageException upon failure
    */
   Blob create(String blob, byte[] content, BlobTargetOption... options) {
```

Comment:
I agree we should add it and default to `application/octet-stream` if `null`. I would place a `DEFAULT_CONTENT_TYPE` constant in `Storage`. I opened #255 for this.